plugins/meson.build

```diff
@@ -28,6 +28,7 @@ subdir('hailuck')
 subdir('intel-spi')
 subdir('iommu')
 subdir('jabra')
+subdir('lenovo-firmware')
```

Comment:
why the `-firmware`? maybe just `lenovo`? or `lenovo-thinklmi` perhaps?

Reply:
I don't want to get in the business of just `lenovo` because what happens when they have some other thing, do they end up with both a `lenovo` and `lenovo-widget` plugin?

I specifically chose `lenovo-firmware` because I think there are other interesting things they could do when they're on their own systems which might not be `thinklmi` based.  But if that's not really the eventual direction I would concede to `lenovo-thinklmi`

Reply:
What's lmi out of interest?

Reply:
If I was gonna guess, "Lenovo Management Interface"?

Reply:
I think it's 'Linux Management Interface', as the team doing it wanted to differentiate it from 'Windows Management Interface' for WMI - at least I think I saw that on a slide deck somewhere :)